### PR TITLE
refactor(part 3): add documentation tool section, improve grammar & add some URLs

### PR DESCRIPTION
**Pull request by lauthieb**
Hello,

_First of all, a huge thank you to all the people who have contributed so far to this wonderful initiative.
Being passionate about UI and standardization, I loved reading the second draft and I want to be part of the contributors by proposing some changes that I hope you will like._

## Proposals

In this PR, I propose you several things:

- to add a documentation tool section because I noticed in the introduction, it was the only one between "design tool" and "translation tool" that didn't have an appropriate section:

CleanShot 2022-07-01 at 01 49 32@2x

- to be more consistent regarding grammar between design, translation & documentation tools
- to align titles & sentences to singular
- to add URLs for design tools

Please, don't hesitate if you have any remarks.

Thanks in advance for your review!

### CODE REVIEW

File: technical-reports/format/terminology.md [hunk 1]
```diff
@@ -26,34 +26,46 @@ For example:
...
+
+## <dfn>Documentation tool</dfn>
+
+A documentation tool is a tool for documenting design tokens usage.
+
+For example:
+
+- [Storybook](https://github.com/salesforce-ux/theo)
+- [Zeroheight](https://amzn.github.io/style-dictionary/)
+- [Backlight](https://backlight.dev/)
```
Comment (kaelig) [inline technical-reports/format/terminology.md:67]: Would Specify, Supernova, and Knapsack also belong to this category?

**Review by kaelig — CHANGES_REQUESTED** [review]
I like the idea of documenting this! There may be overlap of certain tools across multiple categories, but that's okay.

Where do you think a tool like Figma Tokens fits?

File: technical-reports/format/terminology.md [hunk 2]
```diff
@@ -26,34 +26,46 @@ For example:
...
+- ...
+
+## <dfn>Documentation tool</dfn>
+
+A documentation tool is a tool for documenting design tokens usage.
+
+For example:
+
+- [Storybook](https://github.com/salesforce-ux/theo)
+- [Zeroheight](https://amzn.github.io/style-dictionary/)
```
Comment (lauthieb) [inline technical-reports/format/terminology.md:66]:
```suggestion
- [Storybook](https://storybook.js.org/)
- [Zeroheight](https://zeroheight.com)
```

File: technical-reports/format/terminology.md (hunk 1 above)
Comment (lauthieb) [reply technical-reports/format/terminology.md:67]: Yes you're right :)
Just added them!

File: technical-reports/format/terminology.md [hunk 3]
```diff
@@ -26,34 +26,46 @@ For example:
...
+
+## <dfn>Documentation tool</dfn>
+
+A documentation tool is a tool for documenting design tokens usage.
+
+For example:
+
+- [Storybook](https://storybook.js.org/)
+- [Zeroheight](https://zeroheight.com)
+- [Backlight](https://backlight.dev/)
```
Comment (lauthieb) [inline technical-reports/format/terminology.md:67]:
```suggestion
- [Storybook](https://storybook.js.org/)
- [Zeroheight](https://zeroheight.com)
- [Backlight](https://backlight.dev/)
- [Specify](https://specifyapp.com/)
- [Supernova](https://www.supernova.io/)
- [Knapsack](https://www.knapsack.cloud/)
```

**Comment by lauthieb** [comment]
@kaelig, thank you for your review. Just fixed all your suggestions :)

**Review by kaelig — APPROVED** [review]

**Comment by kaelig** [comment]
Merged! Thank you so much for your help ❤️

**Comment by lauthieb** [comment]
@kaelig you're welcome! I enjoyed contribute and can't wait to help you more :)

**Pull request merged**